governance/polls/SparkLend Ethereum - Onboard PT-USDe-27Mar2025 to Morpho Spark DAI Vault and Adjust PT-sUSDe-27Mar2025 Pool Parameters - November 18, 2024.md

```diff
@@ -0,0 +1,57 @@
+---
+title: SparkLend Ethereum - Onboard PT-USDe-27Mar2025 to Morpho Spark DAI Vault and Adjust PT-sUSDe-27Mar2025 Pool Parameters - November 18, 2024
+summary: Signal your support or opposition to 1) onboarding PT-USDe-27Mar2025 to Morpho Spark DAI Vault and 2) adjusting PT-sUSDe-27Mar2025 pool parameters.
+discussion_link: https://forum.sky.money/t/28-nov-2024-proposed-changes-to-spark-for-upcoming-spell/25543
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-11-18T16:00:00
+end_date: 2024-11-21T16:00:00
+---
+# Poll: SparkLend Ethereum - Onboard PT-USDe-27Mar2025 to Morpho Spark DAI Vault and Adjust PT-sUSDe-27Mar2025 Pool Parameters - November 18, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitator. This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9_Weekly_Governance_Cycle-95671ce7-60d8-4d8e-b9b6-de7493560308|0db3) will be active for three days beginning on Monday, November 18 at 16:00 UTC.
+
+**This is a binary vote.**
+
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to onboarding PT-USDe-27Mar2025 to Morpho Spark DAI Vault and to adjusting [PT-sUSDe-27Mar2025](https://morpho.blockanalitica.com/ethereum/markets/5e3e6b1e01c5708055548d82d01db741e37d03b948a7ef9f3d4b962648bcbfa7) pool parameters as described below:
+
+- Onboard PT-USDe-27Mar2025 to Morpho Spark DAI Vault with the configurations that follow: 
```

Comment:
Suggestion: Different headers for onboarding and adjusting current market. Non-binding.